Analytics/Classes/Internal/SEGHTTPClient.h

```diff
@@ -15,6 +15,8 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SEGHTTPClient : NSObject
 
 @property (nonatomic, strong) SEGRequestFactory requestFactory;
+@property (nonatomic, readonly) NSMutableDictionary<NSString *, NSURLSession *> *sessionsByWriteKey;
```

Comment:
The SEGHTTPClient isn't a singleton, so it'll only ever be tied to a single SEGAnalytics instance, and hence only ever talk to a single writeKey?